composer.json

```diff
@@ -55,7 +55,8 @@
         "psr/cache-implementation": "To use metadata caching.",
         "symfony/cache": "To have metadata caching when using Symfony integration.",
         "symfony/config": "To load XML configuration files.",
-        "symfony/twig-bundle": "To use the Swagger UI integration."
+        "symfony/twig-bundle": "To use the Swagger UI integration.",
+        "swagger-api/swagger-ui": "^2.2"
```

Comment:
@teohhanhui Why not keeping the "hijacking" (I like this term too) but displaying an error pointing to the doc explaining that the Swagger UI package and the related Composer script must be installed in order to have the UI? IMO it would be a better DX for people installing the bundle on an existing project (and they can still disable the "hijacking" using the config if they want to).